pkg/postgres/migrations/202411191710_blockNumberFkConstraint/up.go

```diff
@@ -0,0 +1,32 @@
+package _202411191710_blockNumberFkConstraint
+
+import (
+	"database/sql"
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+type Migration struct {
+}
+
+func (m *Migration) Up(db *sql.DB, grm *gorm.DB) error {
+	queries := []string{
+		`alter table operator_directed_reward_submissions add constraint operator_directed_reward_submissions_block_number_fkey foreign key (block_number) references blocks (number) on delete cascade`,
```

Comment:
Rather than adding these as a separate migration, each fk constraint should be added to the table definition when we create it:

```sql
create table if not exists operator_directed_reward_submissions (
    -- stuff....
    CONSTRAINT operator_directed_reward_submissions_block_number_fkey FOREIGN KEY (block_number) REFERENCES blocks(number) ON DELETE CASCADE
)
```

What you did is more appropriate when we're adding constraints after the fact, but since we're creating new tables, it can be added to the table create query

Reply:
Yeah I forgot to add the constraints when I first created the tables.

Reply:
Do you want me to rewrite the migrations here or is there more info for the future?

Reply:
Yeah in this context, updating the migrations is fine. Its only once they make it to master that they shouldnt be changed.

Reply:
Done in [fd2e7b4](https://github.com/Layr-Labs/sidecar/pull/112/commits/fd2e7b42df852c9c636e7310fe0962fdedb6383e)